x-pack/plugins/maps/public/shared/layers/styles/vector_style.js

```diff
@@ -227,7 +227,7 @@ export class VectorStyle extends AbstractStyle {
 
     //scale to [0,1] domain
     for (let i = 0; i < features.length; i++) {
-      const unscaledValue = features[i].properties[fieldName];
+      const unscaledValue = parseFloat(features[i].properties[fieldName]);
       let scaledValue;
       if (typeof  unscaledValue !== 'number' || isNaN(unscaledValue)) {//cannot scale
```

Comment:
You can remove this portion of the condition now: `typeof  unscaledValue !== 'number' || `